packages/salesforcedx-utils-vscode/test/jest/context/workspaceContextUtil.test.ts

```diff
@@ -127,28 +143,48 @@ describe('WorkspaceContext', () => {
     let createConnectionStub: jest.SpyInstance;
 
     beforeEach(() => {
-      createAuthStub = jest
-        .spyOn(AuthInfo, 'create');
-      createConnectionStub = jest
-        .spyOn(Connection, 'create');
+      createAuthStub = jest.spyOn(AuthInfo, 'create');
+      createConnectionStub = jest.spyOn(Connection, 'create');
     });
 
     it('should return connection for the default org', async () => {
       authInfoMock.create.mockResolvedValue(mockAuthInfo as any);
-      authInfoMock.create.mockResolvedValue(mockAuthInfo as any);
-      connectionMock.create.mockResolvedValue(mockConnection as unknown as Promise<Connection<any>>);
+      connectionMock.create.mockResolvedValue(
+        (mockConnection as unknown) as Promise<Connection<any>>
+      );
       const connection = await workspaceContextUtil.getConnection();
-      expect(connectionMock.create).toHaveBeenCalledWith({ authInfo: mockAuthInfo });
+      expect(connectionMock.create).toHaveBeenCalledWith({
+        authInfo: mockAuthInfo
+      });
       expect(connection).toEqual(mockConnection);
     });
 
     it('should return a cached connection for the default org if there is one', async () => {
       authInfoMock.create.mockResolvedValue(mockAuthInfo as any);
-      connectionMock.create.mockResolvedValue(mockConnection as unknown as Promise<Connection<any>>);
+      connectionMock.create.mockResolvedValue(
+        (mockConnection as unknown) as Promise<Connection<any>>
+      );
       await workspaceContextUtil.getConnection();
```

Comment:
my formatter went a little nuts.  We should check if you have prettier running locally in vscode.